src/deluge/gui/colour/rgb.cpp

```diff
@@ -0,0 +1,66 @@
+#include "rgb.h"
+#include "util/functions.h"
+
+RGB RGB::fromHue(int32_t hue) {
+	RGB rgb{};
+	hue = (uint16_t)(hue + 1920) % 192;
```

Comment:
readability: +1920 does nothing when you take it modulus 192

Reply:
Fixed

Reply:
Reverting. Removing the `+ 1920` screws up the color distribution in the keyboard mode and other "rainbow" gradients.

Reply:
Hmm, I think that implies that the conversion to uint16 is over/under flowing

Reply:
Ah right - thanks